code/lib/test/template/stories/before-each.stories.ts

```diff
@@ -0,0 +1,29 @@
+import { global as globalThis } from '@storybook/global';
```

Comment:
Shouldn't we just stop using @storybook/global entirely and just use globalThis directly?
```suggestion
```